eng/pipelines/installer/jobs/base-job.yml

```diff
@@ -431,6 +431,10 @@ jobs:
         displayName: 'Libraries artifacts (AllConfigurations)'
         cleanUnpackFolder: false
 
+  - ${{ if in(parameters.osGroup, 'OSX', 'iOS', 'tvOS') }}:
+    - script: sh $(Build.SourcesDirectory)/eng/install-native-dependencies.sh ${{ parameters.osGroup }}
```

Comment:
you don't need the `sh` do you?

Reply:
*sh would be useful if you need particular shell flavor of if nstall-native-dependencies.sh  does not have executable bit.

Reply:
Addressed in 277c5b5ba66c44a0862a9a536e6aec32df747de9.